src/bin/link_in_test_app.ts

```diff
@@ -15,7 +15,8 @@ fs.writeFileSync(
                 return {
                     ...packageJsonParsed,
                     "main": packageJsonParsed["main"].replace(/^dist\//, ""),
-                    "types": packageJsonParsed["types"].replace(/^dist\//, "")
+                    "types": packageJsonParsed["types"].replace(/^dist\//, ""),
+                    "bin": Object.fromEntries(Object.entries<string>(packageJsonParsed["bin"]).map(([k, v]) => [k, v.replace(/^dist\//, "")]))
```

Comment:
not sure why *nix did not break without this, but on windows it resulted in failures when trying to call `yarn link`

Reply:
Oh very nice thanks!